modules/nlu/src/views/full/intents/SidePanelSection.tsx

```diff
@@ -66,7 +66,7 @@ export const IntentSidePanelSection: FC<Props> = props => {
   const createIntent = async (sanitizedName: string, rawName: string) => {
```

Comment:
Since we don't seem to be using the `rawName` parameter anymore, I think it would be best to remove it here and everywhere we call `createIntent`